force-app/main/default/classes/CQ_GJ_ParentObject_Trigerhandler.cls

```diff
@@ -0,0 +1,68 @@
+public class CQ_GJ_ParentObject_Trigerhandler {
+    
+ 
+    public static void handleBeforeUpdate(List<CQ_GJ_SQX_Parent_Object__c> newParents, Map<Id, CQ_GJ_SQX_Parent_Object__c> oldParentsMap){
+        List<CQ_GJ_SQX_Child_Object__c> parentsToUpdate = new List<CQ_GJ_SQX_Child_Object__c>();
+       Set<Id> parentIds = new Set<Id>();
+        Map<Id, CQ_GJ_SQX_Child_Object__c> parentToChildMap = new Map<Id, CQ_GJ_SQX_Child_Object__c>();
+        
+        for (CQ_GJ_SQX_Parent_Object__c parent : newParents) {
+            parentIds.add(parent.Id);
+            parentToChildMap.put(parent.Id, null);
+        }
+        
+        // Query parent status for all child records outside of the loop
+        for (CQ_GJ_SQX_Child_Object__c child : [SELECT Id, CQ_GJ_SQX_Parent_Info__r.CQ_GJ_Parent_Query__c  
+                                               FROM CQ_GJ_SQX_Child_Object__c 
+                                               WHERE CQ_GJ_SQX_Parent_Info__c IN :parentIds
+                                               AND CQ_GJ_Upload_Marksheet__c != null limit 1]) {
+            parentToChildMap.put(child.CQ_GJ_SQX_Parent_Info__c, child);
+        }
+
```

Comment:
remove this un-necessary log.

Reply:
Oh, i forgot to remove this, 
I will remove this asap